### ISSUE TITLE
Add Tier 3 lines and Poles

### ISSUE THREAD

**Issue by ellingtonisland** [Electrical-Age/ElectricalAge#921]
Electrical age should have Tier 3 Lines and poles added, in real life there are power lines that carry hundreds of thousands to millions of volts. 

since the tier two poles are about 10 blocks high, the tier 3 poles should be like 30 - 40 blocks high or more and the cable should be able to run over several hundred blocks between poles. tier 3 poles should use very high voltage wire instead of the high voltage wire that t1 and t2 poles use.

the transformer for the Tier 3 poles should be about 10 blocks high by 10 blocks wide and should step the 50 - 60kv of Tier 2 up to 300 - 400kv for T3

**Comment by jrddunbr** [comment]
[longterm] and if the solver supports it. Right now T2 are still a somewhat beta feature.

**Comment by AMIDIBOSS** [comment]
30-40 block multiblock devices might cause rendering issues

**Comment by jrddunbr** [comment]
This is true. We already have problems rendering the cables that run between the T2 towers.
